.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+Thanks for submitting your Pull Request!
+
+Please make sure your PR meets the following requirements:
+
+- [ ] Pull Request title is properly formatted: `[WFLY-XYA] Subject`
+- [ ] Pull Request contains link to the JIRA issue
+- [ ] Pull Request contains description of the issue
+- [ ] Pull Request does not include fixes for issues other than the main ticket
+- [ ] Attached commits represent units of work and are properly formatted
+
+For new features, bigger changes and major/minor component upgrades make sure your PR meets also the following requirements:
+
+ - [ ] Pull Request requires a change to the documentation
+ - [ ] Documentation have been updated accordingly
+ - [ ] Tests were added to cover changes
```

Comment:
It is missing:
Analysis was done
Test Plan has been done
Tests were verified in advance